blocks/library/button/style.scss

```diff
@@ -1,31 +1,37 @@
 $blocks-button__height: 46px;
 
 .wp-block-button {
-	display: inline-block;
-	text-decoration: none;
-	margin: 0;
-	padding: 0 24px;
-	height: $blocks-button__height;
-	line-height: $blocks-button__height;
-	cursor: default;
-	border-radius: $blocks-button__height / 2;
-	white-space: nowrap;
-	background-color: $dark-gray-700;
-	vertical-align: top;
-	position: relative;
+	margin-bottom: 1.5em;
 
-	> div {
-		color: $white;
-	}
-
-	a {
+	& .wp-block-button-link {
```

Comment:
Is the nesting necessary here?

Reply:
I think one of the reasons we use the BEM like convention is to avoid nesting as much as we can

Reply:
I think I nested it because several themes had some kind of style conflict. I need to check again, can't remember anymore:)

Reply:
I tested only with couple of themes but noticed some styles overlapping without nesting. I can test more when full time back to work.